serverlessworkflow/modules/ROOT/pages/cloud/deploying-sonataflow-management-console-on-kubernetes.adoc

```diff
@@ -0,0 +1,155 @@
+:keywords: kogito, workflow, quarkus, serverless, kn, oc, openshift, sonataflow
+:table-caption: Data Set
+// envs for common content
+:management_console: SonataFlow Management Console
+:management_console_prefix: sonataflow-management-console
+:command_line_tool: kubectl
+
+
+This document describes how to deploy the {product_name} Management Console on Kubernetes
+
+The SonataFlow Management Console is a web interface designed to manage and monitor workflows. It allows users to initiate workflows, trigger Cloud Events, and monitor their execution.
+
+.Prerequisites
+* Your xref:cloud/operator/build-and-deploy-workflows.adoc[{product_name} application] is deployed and ready to use.
+* A Kubernetes or OpenShift cluster with admin privileges and {command_line_tool} installed.
+* (Optional) xref:security/orchestrating-third-party-services-with-oauth2.adoc[Keycloak server] is installed.
+
+== Set up Kubernetes and deploy your SonataFlow application
+
+The SonataFlow Management Console relies on the underlying services of the SonataFlow application, including the Data Index service. The Data Index provides data from workflow executions, enabling the Management Console to display workflow information. 
+
+Ensure that your Data Index service is deployed and accessible before deploying the Management Console.
+
+.See also:
+. xref:use-cases/advanced-developer-use-cases/deployments/deploying-on-openshift.adoc[Deploying your SonataFlow application on OpenShift ]
+. xref:cloud/operator/supporting-services.adoc[Data Index installation with the Operator]
+
+== (Optional) Deploy Keycloak for production
+
+_If you already have Keycloak deployment or server available you can skip this section._
+
+For production environments, deploy Keycloak to handle authentication or choose your own OAuth2 server. You can refer to the SonataFlow Keycloak documentation xref:security/orchestrating-third-party-services-with-oauth2.adoc[here].
+
+== Deploy {management_console}
+
+Create the deployment for the SonataFlow Management Console by applying the following YAML definition:
+
+.Example configuration in `{management_console_prefix}-deployment.yaml`
+[source,yaml,subs="attributes+"]
+----
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {management_console_prefix}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: {management_console_prefix}
+  template:
+    metadata:
+      labels:
+        app: {management_console_prefix}
+    spec:
+      containers:
+        - name: {management_console_prefix}
+          image: {sonataflow_management_console_imagename}:latest
+          imagePullPolicy: Always
+          ports:
+            - containerPort: 8080
+              name: http
+              protocol: TCP
+          env:
+            - name: SONATAFLOW_MANAGEMENT_CONSOLE_KOGITO_ENV_MODE
+              value: DEV <1>
+            - name: SONATAFLOW_MANAGEMENT_CONSOLE_DATA_INDEX_ENDPOINT
+              value: DATA-INDEX-URL <2>
```

Comment:
```suggestion
              value: publicly.accesible.url.of.data-index-service/graphql <2>
```

not sure how to include this properly, suggesting on

Reply:
Or yea, we have the <2> note in the botton some original one with the suffix should do `DATA_INDEX_URL/graphql`